protected/humhub/modules/user/Events.php

```diff
@@ -2,6 +2,7 @@
 
 namespace humhub\modules\user;
 
+use humhub\modules\content\models\ContentContainer;
```

Comment:
Not to nitpick but this should be placed under `use humhub\modules\user\models\Password;` other than that things look alright to me. :+1: